Arduino_package/hardware/libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -95,14 +90,14 @@ int WiFiClient::available()
         return 0;
     }
     if (_sock >= 0) {
-try_again:
```

Comment:
I would like to suggest to use `_is_blocked` by if condition, to keep both the update and original